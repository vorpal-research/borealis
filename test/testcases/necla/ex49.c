int main( int n){
   int i, sum=0;
   ASSUME( n >= 0);

   for (i=0; i < n; ++i)
      sum = sum +i;

   ASSERT(sum >= 0);
}
