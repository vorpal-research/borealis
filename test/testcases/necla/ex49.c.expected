[
	{
		"location" : 
		{
			"filename" : "test/testcases/necla/ex49.c",
			"loc" : 
			{
				"col" : 4,
				"line" : 10
			}
		},
		"type" : "REQ-01"
	}
]
